utilities/infra.py

```
@@ -681,6 +681,24 @@ def get_all_console_links(console_cli_downloads_spec_links):
+@retry(
+    wait_timeout=TIMEOUT_2MIN,
+    sleep=TIMEOUT_10SEC,
+    exceptions_dict={
+        requests.exceptions.SSLError: [],
+        requests.exceptions.ConnectionError: [],
+        requests.exceptions.Timeout: [],
+    },
+)
+def _download_file(url: str, local_file_name: str) -> str:
+    urllib3.disable_warnings()  # TODO: remove this when we fix the SSL warning
+    response = requests.get(url, verify=False, timeout=TIMEOUT_30SEC)
```

Ruff (0.15.20)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)